resources/tests/k8s/test-job.yml

```diff
@@ -22,3 +22,11 @@ tasks:
       cpu: 100m
       memory: 512M
       gpu: 8
+- id: status
+  type: CheckPod
+  params:
+    refTaskId: job
+    status: Completed
```

Comment:
I think we need to change/add KWOK stages to keep the pods is the Running stage

Reply:
I've upstreamed the support of configurable delays in pod stages to KWOK: https://github.com/kubernetes-sigs/kwok/pull/1037. I thought I've updated you about it.

The default delay of a pod completion(5 second) can be changed via [pod annotations](https://github.com/kubernetes-sigs/kwok/blob/8acd4ce883890b072bcae17b507954218a6c3af4/kustomize/stage/pod/general/pod-complete.yaml#L29).

```
pod-complete.stage.kwok.x-k8s.io/delay
pod-complete.stage.kwok.x-k8s.io/jitterDelay
```

To use the feature, Knavigator must deploy the geenral pod stages in https://github.com/kubernetes-sigs/kwok/tree/main/kustomize/stage/pod/general instead of `stage-fast.yaml`. 

The general stage with delay does's seem in the release. I'll check it with Shiming.

Reply:
Delay is not enough (unless we can specify 'infinity').
Basically, when we start a pod, we want it to be in the Running stage until we delete it.

Reply:
The delay is currently specified in milliseconds (I don't think we need such granularity). We can change the unit to seconds. I think the default maximum value for an integer field in YAML is 999,999. 999,999 seconds is 277 hours, or 21.5 days. Is that good enough?

Reply:
Submitted an issue to KWOK. https://github.com/kubernetes-sigs/kwok/discussions/1070

Reply:
It's not user friendly. It's also not clear why would KWOK complete the pod right after it got started.
There should be an easy way to start the pod and keep it running.

Reply:
My understanding. There are different use cases. Not every one uses KWOK to run tests like Knavigator does.  If a job function test is to ensure all the components work properly, a user may just want it to complete quickly. That's why kwok provides the `stage-fast` option.

Reply:
I discussed with Shiming. YAML should support uint64. So the maximum value is `9223372036854775808 (1<<64-1)`, which is great enough.

 I just tried a delay of `31536000000` (1 year). It seems to be working. So you can just set a very large delay .

```
 pod-complete.stage.kwok.x-k8s.io/delay: "31536000000"
 pod-complete.stage.kwok.x-k8s.io/jitter: "31536000000"
```

Reply:
The general pod stages are not included in the release. Knavigator can deploy it by running 
```console
kubectl kustomize https://github.com/kubernetes-sigs/kwok/kustomize/stage/pod/general
```
or 
```console
kubectl apply -k https://github.com/kubernetes-sigs/kwok/kustomize/stage/pod/general
```